### ISSUE TITLE
Checking status of syncing

### ISSUE THREAD

**Issue by Allanmarbaniang** [astrada/google-drive-ocamlfuse#654]
Is there a way to see the progress of the files syncing between the local drive and Google Drive? Something like a progress bar? I think this is more like a question.

**Comment by strugee** [comment]
No, that's not how this program works. Files aren't "synced". They're made to appear in the local filesystem, but when you access them, they are retrieved, in real time, from the network. Likewise when you write to the mounted filesystem, those writes are uploaded on demand to Google Drive.

**Comment by kevinstsauveur** [comment]
I would like to have this functionnality too! If I am uploading a big file I have edited I want to know the status of the upload.
